### PR TITLE
[WIP] Updating logging.sh test to align with openshift-ansible logging inst…

### PR DESCRIPTION
**Pull request by ewolinetz**
…allation

### CODE REVIEW

**Comment by ewolinetz** [comment]
[test]

**Comment by stevekuznetsov** [comment]
fixes #444

**Comment by stevekuznetsov** [comment]
[test_installer]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
i had `os:cmd` instead of `os::cmd` ...

**Comment by ewolinetz** [comment]
[test]

File: hack/testing/logging.sh [hunk 1]
```diff
@@ -158,14 +158,12 @@ os::start::router
...
 ######### logging specific code starts here ####################
-# not sure how/where this could be created before this . . .
-oc get project logging > /dev/null 2>&1 || os::cmd::expect_success "oadm new-project logging --node-selector=''"
-os::cmd::expect_success "oc project logging > /dev/null"
-
 #initialize logging stack
 source $OS_O_A_L_DIR/hack/testing/init-log-stack
 source $OS_O_A_L_DIR/hack/testing/lib/test-functions
 
+os::cmd::expect_success "oc project logging > /dev/null"
```
Comment (stevekuznetsov) [inline hack/testing/logging.sh]: Won't have output if you don't `$VERBOSE=true` so no need for `/dev/null`

Comment (ewolinetz) [inline hack/testing/logging.sh]: copy pasta'd

**Comment by ewolinetz** [comment]
re[test] so we can get container logs

**Comment by ewolinetz** [comment]
failed on https://github.com/openshift/openshift-ansible/pull/4314

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
flaked... re[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Comment by ewolinetz** [comment]
[test]

**Pull request closed without merging**